docs/UserGuide.md

```diff
@@ -2,15 +2,25 @@
 layout: page
 title: User Guide
 ---
-
-Vax@NUS is a **one stop management app to efficiently track and schedule COVID-19 vaccinations for NUS students.** It is a desktop app **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Vax@NUS can get your appointment management tasks done faster than traditional GUI apps.
-
 * Table of Contents
 {:toc}
 
 :information_source: This icon indicates helpful notes 
 
 --------------------------------------------------------------------------------------------------------------------
+## Introduction
+
+Vax@NUS was conceived at the start of Singapore's COVID-19 vaccination campaign. Our team sought to streamline the 
```

Comment:
Would it be better if we explain how our app keep tracks and schedule for covid-19 vaccinations instead of this?

Reply:
Ok done